blazor/gantt-chart/events.md

```diff
@@ -2258,3 +2258,56 @@ using Syncfusion.Blazor.Gantt
 ```
 
 > We are not going to limit Gantt Chart with these events, we will be adding new events in the future based on the user requests. If the event, you are looking for is not on the list, then please request  [here](https://www.syncfusion.com/feedback/blazor-components).
+
+## OnTaskbarEditing
+
+[OnTaskbarEditing](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Gantt.GanttEvents-1.html#Syncfusion_Blazor_Gantt_GanttEvents_1_TaskbarEditing) event triggers when the taskbar editing begins.
```

Comment:
give more details of taskbar editing actions... like below.
when begining of taskbar editing actions such as taskbar resize, move

Reply:
Modified the content